src/core/component.ts

```diff
@@ -86,6 +86,12 @@ export class Component {
 
     async silentRun(terminal: Terminal, workDir: string, args: string[]): Promise<string> {
         const runTerminal = new StringTerminal();
+        const result = await this.run(runTerminal, workDir, args)
```

Comment:
I think that this part of code must be included between try and catch

Reply:
Right, did it